scheduler/src/cook/kubernetes/api.clj

```diff
@@ -155,7 +155,7 @@
     (.setName metadata (str task-id))
 
     ; container
-    (.setName container "job")
+    (.setName container "required-cook-job-container")
```

Comment:
Let's pull this out into a global variable.

Reply:
Ok.